### PR TITLE
more kubectl doc update

### PR DESCRIPTION
**Pull request by eparis**
new examples section in the docs, rather than part of the long synopsis.
include flags in the man pages.

### CODE REVIEW

File: cmd/gendocs/gen_kubectl_docs.go [hunk 1]
```diff
@@ -68,7 +65,16 @@ func genMarkdown(command *cobra.Command, parent, docsDir string) {
 	fmt.Fprintf(out, "## %s\n\n", name)
 	fmt.Fprintf(out, "%s\n\n", short)
 	fmt.Fprintf(out, "### Synopsis\n\n")
-	fmt.Fprintf(out, "%s\n\n", long)
+	fmt.Fprintf(out, "```\n%s\n```\n\n", long)
```
Comment (vmarmol) [inline cmd/gendocs/gen_kubectl_docs.go]: Why print the synopsis in fixed-width font?

Comment (eparis) [inline cmd/gendocs/gen_kubectl_docs.go]: because md will translate

line one
line two

into 

line one line two

if you dont...

Comment (vmarmol) [inline cmd/gendocs/gen_kubectl_docs.go]: Looks a bit weird in the "rich diff", but it seems fine.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**